api/v1alpha1/release_types_test.go

```diff
@@ -21,310 +21,1070 @@ import (
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gstruct"
 	"github.com/redhat-appstudio/release-service/conditions"
-	"k8s.io/apimachinery/pkg/api/meta"
-	"time"
 
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = Describe("Release type", func() {
-	var release *Release
```

Comment:
Because I don't want releases to be reused for tests in the same context. The alternative would be to reset conditions or keep track of the status of previous tests. We don't want any of those.

Reply:
Yeah, that's what I mean. If you use the higher level `BeforeEach`, that block will be executed once per `Context`, but not once per each test in each context, so whatever you declare there, will be reused within the `Context`.